### DISCUSSION TITLE
How to get my faucet published?

### DISCUSSION THREAD

**Discussion by Grantrocks** [revoxhere/duino-coin#1575] [Q&A]
How can I publish my faucet. I left a email around 5 months ago and no one responded. Sorry for coming here for help but I need to know how to get my app published. Its a faucet its called [Duco Trucco Faucet](https://ducotruccoservices.pythonanywhere.com/).

**Comment by ghost** [comment]
Nice faucet, I claimed the 100 Ducos TY!

**Reply by ghost** [reply]
DM one of the staff members

**Reply by Grantrocks** [reply]
I also made a casino. That's going to be launched today. Check the Duino Coin discord channel at 2.
